vita3k/gui/src/game_selector.cpp

```diff
@@ -178,10 +192,17 @@ void draw_game_selector(GuiState &gui, HostState &host) {
                     }
                     if (ImGui::IsMouseClicked(0))
                         selected[0] = true;
+                    if (gui.title_id != game.title_id)
```

Comment:
no

Reply:
Same here i no want spam change value, slip is better